### PR TITLE
chore(documentation): add missing routes in doc API 2.1

### PR DESCRIPTION
**Pull request by jeremyjaouen**
## Description
Add missing routes in the 2.1 documentation

**Fixes** # MON-5789

## Type of change

- [x] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [ ] Updating documentation (missing information, typo...)

## Target serie

- [ ] 19.10.x
- [ ] 20.04.x
- [ ] 20.10.x
- [x] 21.04.x (master)

### CODE REVIEW

File: doc/API/centreon-api-v2.1.yaml [hunk 1]
```diff
@@ -2689,6 +2689,86 @@ paths:
...
+      summary: 'Delete a Platform'
+      description: "Delete a platform from the topology"
+      responses:
+        '204':
+          description: "Platform deleted"
+        '403':
+          $ref: '#/components/responses/Forbidden'
+        '500':
+          $ref: '#/components/responses/InternalServerError'
+  /gorgone/pollers/{poller_id}/responses/{token}:
```
Comment (sc979) [inline doc/API/centreon-api-v2.1.yaml:2707]: Are you sure, that gorgone endpoints are supported on v2.1 ?
that these routes actually exists in v2.1 conf ?

Comment (jeremyjaouen) [reply doc/API/centreon-api-v2.1.yaml:2707]: Yes

**Review by sc979 — DISMISSED** [review]

File: doc/API/centreon-api-v2.1.yaml [hunk 2]
```diff
@@ -2689,6 +2689,86 @@ paths:
...
+      responses:
+        '200':
+            description: "OK"
+            content:
+              application/json:
+                schema:
+                  type: object
+                  properties:
+                    monitoring_default_downtime_duration:
+                      type: string
```
Comment (kduret) [inline doc/API/centreon-api-v2.1.yaml:2761]:
```suggestion
                      type: integer
```

File: doc/API/centreon-api-v2.1.yaml [hunk 3]
```diff
@@ -2689,6 +2689,86 @@ paths:
...
+                      description: Default downtime duration
+                      example: 3600
+                    monitoring_default_refresh_interval:
+                      type: string
```
Comment (kduret) [inline doc/API/centreon-api-v2.1.yaml:2765]:
```suggestion
                      type: integer
```

**Review by kduret — APPROVED** [review]

**Review by sc979 — APPROVED** [review]

**Pull request merged**